website_sale_one_step_checkout_delivery/controllers/main.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
+from odoo.addons.website_sale_one_step_checkout.controllers.main import WebsiteSale
```

Comment:
Use the controller from website_sale_one_step_checkout because this module extends website_sale_one_step_checkout.